docs/messaging.rst

```diff
@@ -1370,12 +1385,44 @@ handlers should set the parent header and publish status busy / idle,
 just like an execute request.
 
 
-To Do
+Notes
 =====
 
-Missing things include:
+.. _cursor_pos_unicode_note:
+
+``cursor_pos`` and unicode offsets
+----------------------------------
+
+Many frontends, especially those implemented in javascript,
+reported cursor_pos as the interpreter's string index,
+which is not the same as the unicode character offset if the interpreter uses UTF-16 (e.g. javascript or Python 2 on macOS),
+which stores astral-plane characters such as ``𝐚`` as surrogate pairs,
```

Comment:
precise  `U+1D41A` ? Because I'm going to assume somewhere it's not going to render correctly ?